waku/standards/core/64/network.md

```diff
@@ -141,8 +141,8 @@ See the section on [autosharding](#autosharding) for more.
 
 ## RLN rate-limiting
 
-The [17/WAKU2-RLN-RELAY](../17/rln-relay.md) protocol uses [32/RLN-V1](../../../../vac/32/rln-v1.md) proofs
-to ensure that a pre-agreed rate limit is not exceeded by any publisher.
+The [17/WAKU2-RLN-RELAY](../17/rln-relay.md) protocol uses [32/RLN-V1](../../../../vac/32/rln-v1.md), extended with [RLN-V2](https://github.com/vacp2p/rfc-index/blob/main/vac/raw/rln-v2.md) proofs
```

Comment:
Shouldn't this now just be relay "...protocol uses RLN-V2" proofs? In other words, is RLN-V1 still relevant to be mentioned?

Reply:
indeed, rln-v1 is not relevant anymore: fd45bb7495c9123dd2f00becf30643db5660e6e3